src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/ContestedUploadedDocument.java

```diff
@@ -12,6 +13,7 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class ContestedUploadedDocument {
     @JsonProperty("caseDocuments")
```

Comment:
Same for these JsonProperty annotations doesn't seem to be needed here as the attribute is the same as the json property. Although these are all to be migrated to finrem-common soon :)